### PR TITLE
fix whole brain morphologies to load OBJ from nexus

### PR DESCRIPTION
**Pull request by kenjinp**

### CODE REVIEW

File: client/components/MorphologyViewer/index.js [hunk 1]
```diff
@@ -21,8 +21,8 @@ class MorphologyContainer extends React.Component {
     if (morphologySrc) {
       // TODO: refactor after demo
       // if swc, parse
-      const extension = morphologySrc.split('.');
-      if (extension[extension.length - 1] === 'swc' ) {
+      const extension = morphologySrc.split(".");
```
Comment (mdupont) [inline client/components/MorphologyViewer/index.js]: Wouldn't it be better to use String.prototype.endsWith? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith

File: client/components/MorphologyViewer/index.js [hunk 2]
```diff
@@ -67,27 +66,26 @@ class MorphologyContainer extends React.Component {
...
-    let { name, wholeBrain, staticContentLocation } = this.props;
-    const wholeBrainMeshURL = staticContentLocation + "/brain-mesh"
+    let { name, wholeBrain, token } = this.props;
     if (this.viewContainer && morphoData) {
       try {
         this.viewer = new morphoviewer.MorphoViewer(this.viewContainer);
         if (wholeBrain) {
-          this.viewer.addStlToMeshCollection(
+          const wholeBrainMeshURL =
+            "https://bbp.epfl.ch/nexus/v0/data/bbp/atlas/brainparcellationmesh/v0.1.0/39a3078c-1e21-4e1c-9015-82a10ba6797e/attachment";
```
Comment (mdupont) [inline client/components/MorphologyViewer/index.js]: hardcoded URL, is it normal?

Comment (kenjinp) [reply client/components/MorphologyViewer/index.js]: There's only one possible dataset to load at the moment. In the far future there may be more...

Comment (kenjinp) [reply client/components/MorphologyViewer/index.js]: Actually, these things should be resolved in the data somehow, this is a good question for @annakristinkaufmann and @MFSY

**Pull request merged**